server/migrations/20210225201634-ensure-all-threads-have-version-history.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const query = `SELECT * FROM "OffchainThreads" WHERE version_history IS NULL OR version_history='{}'`;
+    const threads = await queryInterface.sequelize.query(query);
+    return Promise.all(threads[0].map(async (thread, i) => {
+      const firstVersion = {
+        timestamp: thread.created_at,
+        body: decodeURIComponent(thread.body)
+      };
+      const update = `UPDATE "OffchainThreads" SET version_history=ARRAY[${firstVersion}] WHERE id='${thread.id}'`;
```

Comment:
Insecure SQL query, this should use a the ORM and a replacement.

Reply:
Yes good to remember... Right now I'm still trying to get a workable stringified json, struggling to escape all the relevant characters in e.g. a Quill text body. This update fails for about 30% of threads, spent all day yesterday trying to debug the various issues preventing a clean data insert. 

Looking for a better high-level strategy in the meantime, unless you have a recommendation for the stringification.

Reply:
(See #checkin for background on blocker)